### PR TITLE
Don't include hidden retired specs in the number of results

### LINKED ISSUE #76: Searches sometimes come up blank
Some race condition on typing into the form produces a search that shows "## results" but doesn't show those results. (Happens for me frequently in Firefox, it seems, but with no predictable reason.)

### PR DESCRIPTION
**Pull request by deniak**
closes #76

### CODE REVIEW

**Comment by Kuzcoo** [comment]
I did not understand the "count" part issue.
I just checkout this branch and it is not working.
May I suggest this fix for the race condition:
```js
  const listElement = document.getElementById("container")
  listElement.addEventListener("transitionend", ({ target }) => {
    if (target.style.opacity === "0") {
      target.style.display = "none"
    }
  })
```
When opacity transition ends we can `display: none` the node.
And removing that part:
```js
      off.forEach((i) => {
        i.style.display = 'none';
      });
```

**Comment by deniak** [comment]
@Kuzcoo Looking at your suggestion, I confirm we aren't talking about the same thing. That PR fixes the count that appears after a search.

I'm guessing you are getting more results than expected but I'm actually having a hard time trying to reproduce the bug. Could you give me a list of steps I can try to reproduce the issue?

**Comment by deniak** [comment]
@Kuzcoo I think I can now reproduce it. Your fix should work but we need to take the retired specs into account. I'll update the PR.

**Comment by Kuzcoo** [comment]
@deniak : for example you can test by typing "web com" in the "Title" input.
If you type char by char you notice that "Introduction to web components" remains, and then disappears. I guess we expect it to stay since the title match the query string "web com".

**Comment by deniak** [comment]
@Kuzcoo I've updated the PR. Can you please try again? If it looks good, I'll redeploy on the main site.

File: mockup1/assets/filtering.js [hunk 1]
```diff
@@ -62,13 +62,10 @@
 
     const total = document.querySelectorAll(SELECTOR_PREFIX);
     const specs = document.querySelectorAll(SELECTOR_PREFIX + '[style*="opacity: 1"]');
-    if (selectors.length > 0)
-      summary.innerHTML = specs.length + ' spec' + (1 === specs.length ? '' : 's') + ' (of ' + (total.length) + ')';
-    else
-      summary.innerHTML = total.length + ' specs (no filters)';
+    summary.innerHTML = specs.length + ' specifications' + (1 === specs.length ? '' : 's') + ' (total of ' + total.length + ' specifications, including retired documents)';
```
Comment (Kuzcoo) [inline mockup1/assets/filtering.js]: extra 's'
also line 66 of the index.html file you  got "specs".
For consistency I think we should use "specifications" too.

Comment (deniak) [reply mockup1/assets/filtering.js]: addressed in https://github.com/w3c/tr-pages/pull/83/commits/ec5600fed4dc767beaf114dc535953ea8aa6db1d

**Comment by deniak** [comment]
@Kuzcoo Thanks for your report and suggestion!

**Comment by Kuzcoo** [comment]
@deniak : very welcome !

**Pull request merged**